testpr-script.js

```diff
@@ -43,9 +43,7 @@ async function test(prId) {
       console.info(stdout);
 
       spawnPromise('git', ['checkout', branchName])
-        .then(() => spawnPromise('yarn', ['install']).then(() => {
-            return spawnPromise('yarn', ['build:deps']);
-          }))
+        .then(() => spawnPromise('yarn', ['build:deps']))
```

Comment:
But what if they _are_ changed? 🤔

Reply:
You will get an error either building when you start the project or when building the deps :) Just as you normally do